pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -257,3 +294,18 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 
 	return insufficientResources
 }
+
+func isIgnoredResource(name string, ignoredResources, ignoredGroups sets.String) bool {
+	if ignoredResources.Has(name) {
+		return true
+	}
+
+	for group := range ignoredGroups {
+		// We know that name is the extended resource, so it is divided by "/".
+		// group matches the substring before "/".
+		if strings.HasPrefix(name, group) {
```

Comment:
`HasPrefix(name, group + "/")`, given that we have validated that the original group doesn't contain slash.